src/service_images/rails/start_shell.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
I think this is slightly different on master. I think we now cd in /cumulus/django or something. Take a look. I think we might be able to just extract this file into service_images directory and they can share it maybe.